packages/client/src/router/index.js

```diff
@@ -103,6 +110,23 @@ export const routes = [
           }
         },
       },
+      {
```

Comment:
Sorry, I missed this in my first review, but these new route definitions are never actually being hit. `vue-router` (in Vue 2, at least) evaluates a URL against the routes in order and uses the first matching route. So the above `/my-grants/:tab` always matches before getting to these more-specific routes. 

If we want the gating and redirecting defined below to work, you'll need to move these two new route definitions above the `/my-grants/:tab` route. 

(Interesting note: If we migrate to Vue 3 soon, that version of vue-router actually has a path ranking algorithm that would have identified your new routes as more specific than the generic tab route, and would have prioritized them regardless of the order. [Example evaluation here](https://paths.esm.dev/?p=AAMeJWyBwRMcaE7AMASAHQxcAe65cxgBAQUIDLahacqWgKDygCkkJhyFpUmYuGwBC5EBWGvd66MAGgB7ia6kY5JsIBEAAA..#).)